src/main/java/ca/ciccc/typinggame/model/WordGenerator.java

```diff
@@ -0,0 +1,9 @@
+package ca.ciccc.typinggame.model;
+
+public class WordGenerator {
+  private String[] wordList;
+
```

Comment:
need constructor pls
It's better if you put some todo comments, for example somethind like belows
`// TODO: read text file to load words list`